src/components/home/FindYourBallot.jsx

```diff
@@ -1,11 +1,75 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import Autosuggest from 'react-autosuggest'
+
+// eslint-disable-next-line import/named
+import { getSuggestions } from 'server/services/arc_gis'
+
 import styles from './find-your-ballot.scss'
 
 export default class FindYourBallot extends React.Component {
+
+  state = {
```

Comment:
I think it would be a bit cleaner to keep all of the state-related logic in the `<FindYourBallotContainer>`